assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -304,10 +312,22 @@ const SelectedVehicleDataLayers = ({
             : stops
         }
       />
-      <RecenterControlWithInterruptibleFollower
-        onUpdate={drawerOffsetAutoCenter}
+      <RecenterControl
+        position="topright"
+        active={followerState.shouldFollow}
+        onActivate={() => {
+          followerState.setShouldFollow(true)
+        }}
+      />
+      <InterruptibleFollower
+        // Do nothing if we don't have a vehicle position yet
+        onUpdate={position.length === 0 ? () => {} : onUpdate}
```

Comment:
**suggestion:** how about making `onUpdate` possibly null / undefined and updating `<Follower/>` accordingly to handle that case (i.e. not call it if undefined)? That feels like it might be a bit more natural than the no-op function approach.

Reply:
Good idea! e8a1c2332a165a24c5bc46bab8dc23b0e80eeb50